### ISSUE TITLE
Default value displayed incorrectly if a function (use @default?)

### ISSUE THREAD

**Issue by morethanreal** [PolymerElements/iron-doc-viewer#30]
``` js
properties: {
  foo: {
    value: function() {
      return this.bar;
    }
  }
}
```

This shows the default value as `undefined.bar`.

Since the function can do anything to set the default value, I tried using the `@default` annotation to set a more descriptive doc, but that does not appear to work.

**Comment by mbleigh** [comment]
I think this is likely a [hydrolysis](https://github.com/polymer/hydrolysis) -- doc viewer just shows what it gets from there

**Comment by morethanreal** [comment]
@garlicnation ?

**Comment by atotic** [comment]
fixed in [May24minorBugFixes cbe976b]
